Podfile

```diff
@@ -132,39 +113,6 @@ abstract_target 'Tools' do
   pod 'SwiftLint', swiftlint_version
 end
 
-# Static Frameworks:
-# ============
-#
-# Make all pods that are not shared across multiple targets into static frameworks by overriding the static_framework? function to return true
-# Linking the shared frameworks statically would lead to duplicate symbols
-# A future version of CocoaPods may make this easier to do. See https://github.com/CocoaPods/CocoaPods/issues/7428
-shared_targets = %w[WordPressFlux]
-dyanmic_framework_pods = %w[WordPressFlux]
-# Statically linking Sentry results in a conflict with `NSDictionary.objectAtKeyPath`, but dynamically
-# linking it resolves this.
-dyanmic_framework_pods += %w[Sentry SentryPrivate]
```

Comment:
The only thing it was affecting – likely unintentionally – is Aztec that was linked statically six targets: apps, and share and draft extensions, increasing the app size. It's not included as a dynamic framework, reducing the app size. 

**Note:** Aztec is also linked statically in Gutenberg, but that's fine and doesn't cause any linker issues.